test/metabase/api/field_test.clj

```diff
@@ -724,3 +725,56 @@
                                     [2 "Small Marble Shoes"]
                                     [3 "Synergistic Granite Chair"]]}
                           (mt/user-http-request :crowberto :get 200 (format "field/%d/values" (mt/id :orders :product_id)))))))))))
+
+(deftest json-unfolding-default-true-test
```

Comment:
I created two tests here, one for when `json-unfolding=true`, and another when `json-unfolding=false`